docker-compose.yml

```diff
@@ -17,11 +9,46 @@ services:
       POSTGRES_PASSWORD: wahlzeit
     ports:
       - 5432:5432
-  # Uncomment for persistence independent of container
-  # volumes:
-  #   - "./pgdata:/var/lib/postgresql/data"
+    networks:
+      - backend
+    # Uncomment for persistence independent of container
+    # volumes:
+    #   - "./pgdata:/var/lib/postgresql/data"
+
   adminer:
     image: adminer
     restart: "no"
     ports:
       - 8090:8080
+
+  backend:
+    image: wahlzeit
+    build:
+      context: .
+    environment:
+      WAHLZEIT_DB_HOST: "db"
+    volumes:
+      - "./src:/src"
```

Comment:
What do we need it for?

Reply:
Thats an hack, as we don't use application.resources - which are natively included inside that jar, but the local file system.

So the docker-container cannot access e.g the inital pictures, config, language conf,  if the file system isn't mounted here

Reply:
Shall i migrate to the native application.resources or leave it as it is?

Reply:
Ah I get it! How much effort do you think it would be to migrate?

Reply:
I will just do the migration - it's worth it's time!

Reply:
awesome, thx for your engagement!

Reply:
@georg-schwarz  resources now working .

Regular directory file iteration is broken now, so the initial photos paths are hardcoded now - see: `SysConfig`

Reply:
added in bc1ca02